python_cdk/src/detectors/python-cdk-sagemaker-notebook-missing-encryption/python-cdk-sagemaker-notebook-missing-encryption_compliant.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-cdk-sagemaker-notebook-missing-encryption@v1.0 defects=0}
+import aws_cdk as cdk
+from aws_cdk.aws_iam import Role, ServicePrincipal
+from aws_cdk.aws_sagemaker import CfnNotebookInstance
+
+class CdkStarterself(cdk.self):
+  def __init__(self, scope: cdk.App, id: str):
+    super(scope, id)
+
+    # Compliant: `kms_key_id` is set.
```

Comment:
"# Compliant: The CfnNotebookInstance instantiation sets the kms_key_id property, enabling encryption with the specified KMS key."